pyro/contrib/epidemiology/sir.py

```diff
@@ -10,26 +10,35 @@
 from .compartmental import CompartmentalModel
 
 
-class SIRModel(CompartmentalModel):
+class SimpleSIRModel(CompartmentalModel):
     """
     Susceptible-Infected-Recovered model.
 
-    :param int population:
-    :param float recovery_time:
+    To customize this model we recommend forking and editing this class.
+
+    This is a stochastic discrete-time discrete-state model with three
+    compartments: "S" for susceptible, "I" for infected, and "R" for
+    recovered individuals (the recovered individuals are implicit: ``R =
+    population - S - I``) with transitions ``S -> I -> R``.
+
+    :param int population: Total ``population = S + I + R``.
+    :param float recovery_time: Mean recovery time (duration in state
+        ``I``). Must be greater than 1.
     :param iterable data: Time series of new observed infections.
     :param int data: Time series of new observed infections.
-    :param int num_quant_bins: The number of quantization bins to use. Note that
-            computational cost is exponential in `num_quant_bins`. Defaults to 4.
+    :param int num_quant_bins: The number of quantization bins to use.
+        Note that computational cost is exponential in `num_quant_bins`.
+        Defaults to 4.
     """
 
     def __init__(self, population, recovery_time, data, *,
                  num_quant_bins=4):
```

Comment:
i don't see where `num_quant_bins` is piped here?

Reply:
oops, thanks for catching, I've removed.